packages/jest-haste-map/src/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -17,5 +17,5 @@ exports[`HasteMap warns on duplicate module ids 1`] = `
   Duplicate module name: Strawberry
   Paths: /fruits/raspberry.js collides with /fruits/strawberry.js
 
-This warning is caused by a @providesModule declaration with the same name across two different files."
+ This warning is caused by a @providesModule declaration with the same name across two different files."
```

Comment:
I think this is a leftover from your previous commit. You should try not to have any merge conflicts but rather do `git pull --rebase upstream master`. This way you don't create merge commits that contain changes from previous commits. Please get rid of this whitespace again.